### DISCUSSION TITLE
Autocompletion for PHP Attributes

### DISCUSSION THREAD

**Discussion by ghost** [apache/netbeans#4296] [General]
Hello,

Just a simple question. Is it possible to activate autocompletion for
PHP Attributes in Netbeans 14 ?

For an attribute like this :
```
#[Attribute]
final class MaxLength(int $maxLength) { ... }
```
And use it like this :
```
#[MaxLength(40)] // <-- Autocomplete here !
private string $test
```

I didn't find any documentation on this trick.

Thank you !

Sébastien

**Comment by KacerCZ** [comment]
Related issue: https://github.com/apache/netbeans/issues/4300
